### ISSUE TITLE
phantomjs2 is broken

### ISSUE THREAD

**Issue by Lassulus** [NixOS/nixpkgs#29144]
## Issue description

on current master run `nix-shell -p phantomjs2 --run 'phantomjs --version'`

output is
```
This application failed to start because it could not find or load the Qt platform plugin "xcb"
in "".

Reinstalling the application may fix this problem.
PhantomJS has crashed. Please read the bug reporting guide at
<http://phantomjs.org/bug-reporting.html> and file a bug report.
Aborted
```

this is also breaking pkgs.fast-cli
https://github.com/NixOS/nixpkgs/issues/28643

## Technical details

* Nix version: 1.11.14
* Nixpkgs version: 5d1a8ecd916893ac377311de4abd5d7043646280
* Sandboxing enabled: yes

**Comment by 7c6f434c** [comment]
A upstream PhantomJS doesn't have a maintainer, as the last one has stepped down after the official release of Headless Chrome with Chrome Driver…

**Comment by 7c6f434c** [comment]
As a workaround, adding the `qt5.qtbase` package's `/bin/` subdirectory to `$PATH` seems to help here.

**Comment by orivej** [comment]
Fixed by #30036
